Geometry/MTDGeometryBuilder/src/MTDParametersFromDD.cc

```diff
@@ -55,6 +57,50 @@ bool MTDParametersFromDD::build(const DDCompactView* cvp, PMTDParameters& ptp) {
   return true;
 }
 
+bool MTDParametersFromDD::build(const cms::DDCompactView* cvp, PMTDParameters& ptp) {
+  cms::DDVectorsMap vmap = cvp->detector()->vectors();
+
+  std::array<std::string, 2> mtdSubdet{{"BTL", "ETL"}};
+  int subdet(0);
+  for (const auto& name : mtdSubdet) {
+    std::vector<double> tmpPars;
```

Comment:
this is not used?

Reply:
@kpedro88 no, a left-over of code adaptation to be removed, thanks for catching it